src/actions/events/events-actions.js

```diff
@@ -0,0 +1,114 @@
+import axios from 'axios'
+
+const baseUrl = 'http://localhost:4000'
```

Comment:
with axios you can create an instance in another file where you set the baseUrl and then import axios from that file from anywhere, try it out!
```
// src/axios.js
import axios from "axios";

const instance = axios.create({
  baseURL: "http://localhost:4000"
});
export default instance;
```

Reply:
How do I now use this then in my actions? Because then baseUrl is not defined. Or do I need to import instance in the action and use that instead of post(`${baseUrl}/cuttings` ?

Reply:
the in your actions you use it like
```
import axios from '../axios.js'

axios.post(/cuttings, { body })
```